boot/zephyr/boards/nrf52840_pca10056.conf

```diff
@@ -0,0 +1,30 @@
+# The UART is used for Serial Recovery, so logging requires
```

Comment:
Please remove this file. We use UART serial recovery, no-multithreading for smaller footprint which is configured well via kconfig file.

Reply:
OK. 
Ill close the PR then. No point in merging without that file.